docs/development/go/go-structures/index.md

```diff
@@ -18,235 +19,532 @@ external_resources:
+
+- When using a struct literal in this way, you do not need to specify all of the fields, as shown on line 18. Because the employeeID for `heather` was not defined, it takes on the zero value (for an integer, this is `0`).
+
+- Lastly, you can also use a struct literal without listing the fields' names, as shown on line 21. The values for the fields will be assigned according to the order that the fields are defined in the struct type definition. You must supply values for all of the fields in order to use this syntax.
+
+    {{< note >}}
+The `mihalis` variable is defined using the `:=` syntax, which infers the `Employee` type for the variable from the assigned value.
 {{< /note >}}
 
-The output of `structures.go` will be the following:
+### Comparing Structs
+
+Structs can be compared for equality. Two structs are equal if they have the same type and if their fields' values are equal.
+
+{{< file "employee.go" go >}}
+package main
+
+import (
+    "fmt"
+)
+
+type Employee struct {
+    FirstName string
+    employeeID int
+}
+
+func main() {
+    employee1 := Employee{"Heather", 1910234}
+    employee2 := Employee{"Heather", 1910234}
+    fmt.Println(employee1 == employee2)
+}
+{{< /file >}}
+
+The output of `employee.go` will be:
+
+    go run employee.go
 
-    go run structures.go
 {{< output >}}
-0 0
-{23 Hello! -2}
-{0 Hi! 12}
-p: [{0 Hi! 12} {0  0} {23 Hello! -2}]
-p2: {1 Two 3.1}
-p: [{0 Hi! 12} {0  0} {23 Hello! -2}]
+true
 {{< /output >}}
 
 {{< note >}}
-Structures, in particular, and Go types, in general, are usually defined outside the `main()` function in order to have a global scope and be available to the entire Go package, unless you want to clarify that a type is only useful within the current scope and is not expected to be used elsewhere in your code.
+Structs cannot be ordered with operators like greater-than  `>` or less-than `<`.
 {{< /note >}}
 
-## Pointers and Structures
+### Accessing Fields
 
-As Go supports pointers, you can create pointers to structures. The use of pointer structures is illustrated in `pointers.go`.
+You can access a specific field using the struct variable name followed by a `.` character followed by the name of the field (also referred to as *dot notation*). Given an `Employee` variable named `mihalis`, the struct's two fields can be individually accessed as `mihalis.FirstName` and `mihalis.employeeID`:
 
-{{< file "./pointers.go" go >}}
+{{< file "employee.go" go >}}
 package main
 
 import (
     "fmt"
 )
 
-type myStructure struct {
-    Name    string
-    Surname string
-    Height  int32
+type Employee struct {
+    FirstName string
+    employeeID int
 }
 
 func main() {
-    ptr := &myStructure{"Mihalis", "Pointer", 123}
-    s2 := myStructure{"Mihalis", "s2", 123}
-
-    fmt.Println("Getting a specific structure field:", (*ptr).Surname)
-    fmt.Println(s2.Name, s2.Surname)
-    fmt.Println("As a pointer:", ptr)
-    fmt.Println("Memory address:", &ptr)
```

Comment:
This line (`fmt.Println("Memory address:", &ptr)`) gets the memory address of the pointer, not of the object at the pointer (e.g. the pointer's value itself), which I thought was a bit confusing